src/test/java/seedu/address/model/student/LessonTimeTest.java

```diff
@@ -94,7 +94,29 @@ public void canMerge() {
         assertFalse(lt.canMerge(new LessonTime("SUN-14:00-15:30")));
     }
 
+    @Test
+    public void compareTo() {
+        LessonTime lt = new LessonTime("SUN-11:00-13:00");
+
+        //Same value -> 0
+        assertEquals(lt.compareTo(lt), 0);
+
+        //LessonTime same day, earlier start -> 1
```

Comment:
Can add another test to compare against a different LessonTime object with the exact same values? Also maybe another for boundary times (2359 and 0000)